src/basic.h

```diff
@@ -29,7 +29,7 @@ namespace SymEngine {
 
 class Visitor;
 class Symbol;
-
+//class Polynomial;
```

Comment:
I'll see to it that it is removed in the next commit. Any other clean-up you find necessary here, please let me know.
cc @sushant-hiray @shivamvats